### PR TITLE
SEO-2475_Site_Audit_Issues

### PR DESCRIPTION
**Pull request by gowthamanthavasiyappan**
Hi @Aishwarya-Ganesan,

Worked on site audit issues, merge these changes,

Regards,
Gowthaman

### CODE REVIEW

File: PHP/AutoComplete/Getting-Started.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-description: getting started
+title: Getting Started with PHP AutoComplete Control | Syncfusion
+description: Learn here about getting started with Syncfusion Essential PHP AutoComplete Control, its elements, and more.
 platform: php
 control: Autocomplete 
 documentation: ug
 ---
 
-#Overview
+# Overview of PHP AutoComplete
```
Comment (Aishwarya-Ganesan) [inline PHP/AutoComplete/Getting-Started.md:10]: Getting started or overview?

File: PHP/AutoComplete/Getting-Started.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
 The PHP AutoComplete control is a textbox control that provides a list of suggestions based on your query. When you enter text into the text box, the control performs a search operation and provides a list of results. There are several filter types available to perform the search.
 
-# Getting Started
```
Comment (Aishwarya-Ganesan) [inline PHP/AutoComplete/Getting-Started.md:14]: why Overview & Getting started in same page?

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented.

File: PHP/AutoComplete/Getting-Started.md (hunk 1 above)
Comment (gowthamanthavasiyappan) [reply PHP/AutoComplete/Getting-Started.md:10]: Getting Started, changed now

File: PHP/AutoComplete/Getting-Started.md (hunk 2 above)
Comment (gowthamanthavasiyappan) [reply PHP/AutoComplete/Getting-Started.md:14]: Removed Overview

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are Fine.

**Pull request merged**